lightspark_wallet/lib/src/objects/amazon_s3_funds_recovery_kit.dart

```diff
@@ -1,37 +1,44 @@
+
 // Copyright ©, 2023-present, Lightspark Group, Inc. - All Rights Reserved
 
 import './funds_recovery_kit.dart';
 
+
+
 class AmazonS3FundsRecoveryKit implements FundsRecoveryKit {
-  /// The bitcoin address where the funds should be sent if the recovery kit is used.
-  @override
-  final String bitcoinWalletAddress;
 
-  /// The URL of the Amazon S3 bucket URL where we should upload the funds recovery kit.
-  final String s3BucketUrl;
+    /// The bitcoin address where the funds should be sent if the recovery kit is used.
+@override
+final String bitcoinWalletAddress;
```

Comment:
Looks like formatting is broken. You may need to install the formatter (just flutter/dart) - https://docs.flutter.dev/get-started/install

In this case, it's super unlikely you'll actually need this in the future for a while, so I'm just running it for you.